workflow/scripts/get_genome.py

```
@@ -0,0 +1,159 @@
+        error += [
+            "The result from fetching NCBI genome data has zero length. Please check your internet connection!"
+        ]
+    else:
+        ncbi_genome = [
+            i.split("\t")
+            for i in ncbi_result.split("\n")
+            if not (i.startswith("New version") or i.startswith("Warning"))
+        ]
+        ncbi_genome = dict(zip(ncbi_genome[0], ncbi_genome[1]))
+        log += ["Found the following genome(s):"]
+        for k in ncbi_genome.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)